test/mlir/test/gc/Transforms/GPU/linalg-to-xegpu-dpas.mlir

```diff
@@ -63,7 +63,7 @@ func.func @matmul(%arg0: memref<32x32xf16>, %arg1: memref<32x32xf16>, %arg2: mem
 
 // Extract DPAS-sized chunks from larger loaded tile A.
 // Tile B is already in the correct shape.
-// CHECK:   %[[vA_flat:.+]] = vector.shape_cast %[[vA]] : vector<32x8x2xf16> to vector<512xf16>
+// CHECK:   %[[vA_flat:.+]] = vector.shape_cast %[[vA]] : vector<32x16xf16> to vector<512xf16>
```

Comment:
we do not load the `A` matrix via `vnni_axis=1` anymore (see [`packed_attr`](https://github.com/intel/graph-compiler/pull/201#discussion_r1698662219))